docs/reference-guide/modules/events/pages/event-processors/streaming.adoc

```diff
@@ -1101,9 +836,8 @@ By default, Axon components will use the `SequentialPerAggregatePolicy`, making
 Check out <<sequential-processing,this>> section to understand how to influence the sequencing policy.
```

Comment:
This sequencing policy section notes nothing of the new `@SequencingPolicy` annotation that can be used per event and class. Furthermore, it still talks about the `SequentialPerAggregatePolicy`, which is simply not true as that's not an option for DCB. This section needs a more rigorous rewrite for sure!